R/plot.R

```diff
@@ -111,58 +111,51 @@ plot_ethnicity <- function(x) {
 plot_visit_profile <- function(x) {
   x %>%
     mutate_at(vars(visit_start_datetime), ~ as.Date(.)) %>%
+    # Shouldn't really hardcode these dates. Will edit as we set tolerance for data quality report.
+    filter(visit_start_datetime > "2017-01-01" & visit_start_datetime <= Sys.Date()) %>% 
```

Comment:
Why is there still a hardcoded date here

Reply:
Modified to user given start and end dates